src/Containers/Profile/Profile.scss

```diff
@@ -0,0 +1,194 @@
+@import '../../assets/sass/global.scss';
+#profile {
+  input,
+  button {
+    width: 80%;
+  }
+  button {
+    width: 80%;
+  }
+  label {
+    color: #444;
+  }
+  textarea {
+    border: 2px solid #efefef;
+    margin-bottom: 10px;
+    padding: 0 6px;
+    width: 90%;
+    background: #fafafa;
+    &:focus {
+      outline: none !important;
+      border: 2px solid rgba(#aca9a9, 0.5);
+      box-shadow: 2px 0 6px rgba(#efefef, 0.8);
+    }
+  }
+  .navbar {
+    background-color: white;
+  }
+}
+
+.body {
```

Comment:
Hey I see here that you are editing body directly this will have effects on every page, I will suggest that you add padding in your `#profile` instead